packages/@uppy/svelte/src/components/StatusBar.svelte

```diff
@@ -17,10 +20,10 @@
       target: container
     }
 
-    uppy.use(StatusBarPlugin, options);
-    plugin = uppy.getPlugin(options.id) as StatusBarPlugin;
+    uppy.use(StatusBarPlugin as new(uppy: any, opts: typeof options) => StatusBarPlugin<M, B>, options);
```

Comment:
This is a bit concerning, if users need to type that to consume our plugins, it's kind of a deal breaker.